src/components/shared/Backtrace.tsx

```diff
@@ -22,44 +22,74 @@ type Props = {
   readonly stackIndex: IndexIntoStackTable;
   readonly implementationFilter: ImplementationFilter;
   readonly categories: CategoryList;
+  readonly onStackFrameClick?: (stackIndex: IndexIntoStackTable) => void;
 };
 
 export function Backtrace(props: Props) {
-  const { stackIndex, thread, implementationFilter, maxStacks, categories } =
-    props;
+  const {
+    stackIndex,
+    thread,
+    implementationFilter,
+    maxStacks,
+    categories,
+    onStackFrameClick,
+  } = props;
   const funcNamesAndOrigins = getBacktraceItemsForStack(
     stackIndex,
     implementationFilter,
     thread
   );
 
+  // Build a mapping from filtered frame index to stack index
+  const { stackTable } = thread;
+  const stackIndices: IndexIntoStackTable[] = [];
```

Comment:
We've mostly aligned on using "indexes" instead of "indices" for the plural of index in this codebase.